src/commands/publish.rs

```diff
@@ -25,11 +25,31 @@ pub struct PublishOutput {
 pub fn publish(
     user: &GlobalUser,
     target: &mut Target,
-    deploy_config: DeployConfig,
+    deployments: DeploymentSet,
     out: Output,
 ) -> Result<(), failure::Error> {
     validate_target_required_fields_present(target)?;
 
+    let deploy = |target: &Target| match deploy::worker(&user, &deployments) {
+        Ok(urls) => {
+            let result_msg = if !urls.is_empty() {
+                format!("Successfully published your script to {}", urls[0])
+            } else {
+                "Successfully published your script".to_owned()
```

Comment:
i assume this is the cron only response? it would be good for this output to include feedback on crons published as well, i think; if possible next approximate scheduled time or something similar. if that detail isn't possible, you can repeat back the cron schedule that was set.

to me this suggests that the deploy worker function result should perhaps become a struct with `urls` and `crons` or `schedules` to facilitate this, which would turn this into a kind of list, and in the case of json output, would add another field (`urls` and `schedules`)